massa-factory-worker/src/tests/tools.rs

```diff
@@ -38,9 +26,6 @@ use massa_wallet::test_exports::create_test_wallet;
 /// Then you can use the method `get_next_created_block` that will manage the answers from the mock to the factory depending on the parameters you gave.
```

Comment:
You removed the get_next_created_block method so remove it in doc comment here too

Reply:
Done.